### PR TITLE
tests(control-flow): add a test onPrepare to make sure control flow does...

### LINKED ISSUE #1385: WebDriver control flow can sometimes get locked

### PR DESCRIPTION
**Pull request by juliemr**
...n't lock

Closes #1385

### CODE REVIEW

File: spec/controlLockConf.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+var env = require('./environment.js');
+var webdriver = require('selenium-webdriver');
+var portProber = require('selenium-webdriver/net/portprober');
```
Comment (hankduan) [inline spec/controlLockConf.js]: Why do you need this?

**Comment by hankduan** [comment]
LGTM

File: spec/controlLockConf.js (hunk 1 above)
Comment (juliemr) [inline spec/controlLockConf.js]: From an old version - removed.

**Comment by juliemr** [comment]
Merged as 3b03ab2acf3d8666722831a46197c95772a4d16c

**Pull request closed without merging**